src/runtime/button-api.js

```diff
@@ -117,6 +122,12 @@ export class ButtonApi {
     }
     button.setAttribute('title', msg(TITLE_LANG_MAP, button) || '');
     button.addEventListener('click', callback);
+    button.addEventListener('click', () =>
+      this.eventManager_.logSwgEvent(
+        AnalyticsEvent.ACTION_SWG_BUTTON_CLICK,
+        true
```

Comment:
maybe a little comment on what this true is for //From user Action

Reply:
Done.